lib/pengine/unpack.c

```diff
@@ -4245,6 +4233,10 @@ add_node_attrs(xmlNode *xml_obj, pe_node_t *node, bool overwrite,
     pe__unpack_dataset_nvpairs(xml_obj, XML_TAG_ATTR_SETS, &rule_data,
                                node->details->attrs, NULL, overwrite, data_set);
 
+    pe__unpack_dataset_nvpairs(xml_obj, XML_TAG_UTILIZATION, &rule_data,
+                               node->details->utilization, NULL,
+                               FALSE, data_set);
+
```

Comment:
We should add a scheduler regression test for this. It should be sufficient to take one of the existing utilization tests and make one of the node attributes transient.

Reply:
Added.